### ISSUE TITLE
[Feature Request] Modded Biomes compatible with Beach Crab Rave!

### ISSUE THREAD

**Issue by DEAFG0AT** [mosharky/GroundedOrigins#20]
**Description**
Mods like Regions Unexplored, Oh The Biomes You'll Go, and Terralith add many knew biomes that would would consider a beach (ex: tropics from regions unexplored, gravel beach from terralith, etc.). However, the Beach Crab origin is only able to activate its Rave! ability on (I believe) the default vanilla beach biomes. Is there any way the mod could accept a modded biome with a beach-related tag? Thanks!

**Comment by mosharky** [comment]
yeah I'll change it to use biome tags next update ;)

**Comment by DEAFG0AT** [comment]
Thanks! Do you think that'll be backported to 1.19.2 at any point?

**Comment by mosharky** [comment]
yup! I'm planning to do a small update for 1.19.2 and do a major overhaul of the mod for 1.20. I'm also going to be exclusively updating my mod on Modrinth.

**Comment by DEAFG0AT** [comment]
That's great! I switched over to Modrinth from Curseforge a couple of months ago and it's so much easier. All of my mods are hosted there exclusively :D
I just wish more people posted their mods there!

**Comment by mosharky** [comment]
yup i'm loving modrinth, hope it becomes the main platform everyone uses
